### ISSUE TITLE
How to get html-loader to process HTML's script's src?

### ISSUE THREAD

**Issue by dreadcast** [webpack-contrib/html-loader#196]
**I'm submitting a bug report**
Question was asked [here](https://stackoverflow.com/questions/54594114/how-to-get-html-loader-to-process-htmls-scripts-src)

**Webpack version:**
4.28.2

**HTML-Loader version:**
0.5.5

**Please tell us about your environment:**
OSX 10.14.2

**Current behavior:**
script's `src` attribute outputs `[object Object]`

**Expected/desired behavior:**
script's `src` attribute outputs `/path/to/bundle.js`

* **How to reproduce** 
Fork [this](https://github.com/dreadcast/html-loader) to reproduce.

* **Browser:** all

* **Language:** Javascript

**Comment by JakubKoralewski** [comment]
Getting the same thing with link:href:

```html
<link rel="stylesheet" href="./scss/index.scss">
```

turns into:
```html
<link rel=stylesheet href=[object Object]>
```

with config:
```javascript
module: {
        rules: [
            {
                test: /\.html$/,
                use: [
                    'file-loader?name=[name].[ext]', 'extract-loader',
                    {
                        loader: 'html-loader',
                        options: {
                            conservativeCollapse: false,
                            minimize: true,
                            attrs: ['link:href'],
                        },
                    },
                ],
            },
            {
                test: /\.js$/,
                exclude: /node-modules/,
                use: 'babel-loader',
            },

            {
                test: /\.scss$/,
                use: [
                    MiniCssExtractPlugin.loader,
                    'css-loader',
                    {
                        loader: 'sass-loader',
                        options: {minimize: true},
                    },
                    'postcss-loader',
                ],
            },
        ],
    },
```

versions:
html-loader: 0.5.5
extract-loader: 3.1.0
file-loader: 4.2.0 🚬

**Comment by nrueckmann** [comment]
@JakubKoralewski try to change the order of your loaders for `scss` files. `sass-loader` needs to run first (being the last element in array).

**Comment by dipeshba** [comment]
@evilebottnawi Is this really fixed? I am using hemlt-loader 1.1.0 but i am still getting same issue.

**Comment by evilebottnawi** [comment]
No need duplicate, it is create a spam
